.circleci/run-tests.sh

```diff
@@ -100,6 +100,9 @@ case $1 in
     chipyard-sodor)
         run_asm ${mapping[$1]}
         ;;
+    chipyard-ibex)
+        make run-binary-fast -C $LOCAL_SIM_DIR ${mapping[$1]} BINARY=$RISCV/riscv64-unknown-elf/share/riscv-tests/benchmarks/multiply.riscv
```

Comment:
Should this instead use a 32b based test? IIRC these tests are compiled for 64b platforms by default.